webapp/src/components/link_tooltip/link_tooltip.jsx

```diff
@@ -97,7 +105,7 @@ export const LinkTooltip = ({href, connected, show, theme}) => {
                             {data.repo}
                         </span>
                         {' on '}
-                        <span>{date}</span>
+                        <span>{date + ' by ' + userName}</span>
```

Comment:
Should the word `by` be shown if the `userName` is empty for what ever reason?

Reply:
Updated!